python/lsst/cp/pipe/utils.py

```diff
@@ -315,6 +315,8 @@ def irlsFit(initialParams, dataX, dataY, function, weightsY=None):
         Function to fit.
     weightsY : `numpy.array` [`float`]
         Weights to apply to the data.
+    weightType : `str`, optional
```

Comment:
Should you say here (or above) that the list of options for weights is explained in the body of the function? or do you formally need to exhaustively list all the options here in the docstring?

Reply:
I've listed all the options here, because it seemed reasonable.  I've also added a raise if an unknown option is passed.